acme/setup.py

```diff
@@ -53,7 +53,7 @@
     version=version,
     description='ACME protocol implementation in Python',
     url='https://github.com/letsencrypt/letsencrypt',
-    author="Let's Encrypt Project",
+    author="Electronic Frontier Foundation",
```

Comment:
that doesn't look correct - previous version captured authors other than EFF. Why not "Certbot Project"? Frankly speaking, it was Let's Encrypt project who authored it so far ;)